src/components/inline_edit/inline_edit_form.tsx

```diff
@@ -125,6 +146,9 @@ export const EuiInlineEditForm: FunctionComponent<EuiInlineEditFormProps> = ({
   children,
   sizes,
   defaultValue,
+  value: controlledValue,
+  onChange,
+  onCancel,
   placeholder,
```

Comment:
See above comments - setting the undefined fallbacks to `''` satisfies typescript string requirements while still passing any falsy checks.
```suggestion
  defaultValue = '',
  value: controlledValue = '',
  onChange,
  onCancel,
  placeholder = '',
```